### PR TITLE
chore: libwxgtk3.0-gtk3-dev with libwxgtk3.2-dev for Ubuntu 24.04

### PR DESCRIPTION
**Pull request by nakajima-yuusuke**
**What I Did**

・Replaced libwxgtk3.0-gtk3-dev with libwxgtk3.2-dev for compatibility with Ubuntu 24.04 (Noble).
・Implemented logic to dynamically select the appropriate libwxgtk package based on the Ubuntu version (jammy or noble) to maintain compatibility across both environments.

**Background, Purpose, and Context**

・The CI environment is transitioning from Ubuntu 22.04 (Jammy) to Ubuntu 24.04 (Noble).
・In Ubuntu 24.04, libwxgtk3.0-gtk3-dev is no longer supported, and libwxgtk3.2-dev is required for compatibility.
・To ensure the setup script works seamlessly during the transition period, the implementation dynamically selects the appropriate package (libwxgtk3.0-gtk3-dev for Jammy, libwxgtk3.2-dev for Noble) based on the detected Ubuntu version.
・This change ensures compatibility with both Ubuntu versions until the full migration to Noble is complete.

### CODE REVIEW

File: src/linux.js [hunk 1]
```diff
@@ -1,16 +1,31 @@
...
+    listeners: {
+      stdout: (data) => {
+        ubuntuVersion += data.toString().trim();
+      }
+    }
+  });
+
+  const wxgtkPackage = (ubuntuVersion === 'jammy')
+    ? 'libwxgtk3.2-dev'
+    : 'libwxgtk3.0-gtk3-dev';
```
Comment (hidakatsuya) [inline src/linux.js:25]: It seems the condition is reversed. I believe the correct code should be: 
```suggestion
  const wxgtkPackage = (ubuntuVersion === 'jammy')
    ? 'libwxgtk3.0-gtk3-dev'
    : 'libwxgtk3.2-dev';
```
Would you mind confirming?

**Review by hidakatsuya — COMMENTED** [review]
Thank you for your contribution. I’ve left a comment on the code. Please take a look.

File: src/linux.js (hunk 1 above)
Comment (nakajima-yuusuke) [reply src/linux.js]: Hi @hidakatsuya ,

Thank you for pointing out the reversed condition and sorry for the delayed response. 
I’ve just updated the code according to your suggestion.

**Comment by nakajima-yuusuke** [comment]
I pushed the changes to the repository and would really appreciate it if you could review them again and trigger the CI. Let me know if there's anything else I can do. 

Thanks again for your help.

**Review by hidakatsuya — APPROVED** [review]

**Comment by hidakatsuya** [comment]
Merged. I appreciate your contribution.

**Comment by nakajima-yuusuke** [comment]
Thank you for reviewing and merging the changes! I appreciate your time and support throughout this process. Please let me know if there’s anything else I can help with.

**Pull request merged**